applications/FluidDynamicsApplication/python_scripts/navier_stokes_base_solver.py

```diff
@@ -248,3 +151,30 @@ def _GetAutomaticTimeSteppingUtility(self):
                                                                      self.settings["time_stepping"])
 
         return EstimateDeltaTimeUtility
+
+    def _replace_elements_and_conditions(self):
+        ## Get element number of nodes and domain size
+        domain_size = self.main_model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE]
+        elem_num_nodes = len(self.main_model_part.Elements.__iter__().__next__().GetNodes())
```

Comment:
not sure what is being done here...

Reply:
At the end, it returns the first iterator of the Elements container.